docs/installingNative.md

```diff
@@ -130,43 +119,48 @@ Now you can install the other dependencies.
   cd PrairieLearn
   ```
 
-!!! note "Setup a venv"
+- Set up a Python virtual environment:
 
-    It is recommended to use a virtual environment for Python dependencies. You can create a virtual environment within PrairieLearn:
-
-    === "Native"
+  === "uv"
 
         ```sh
-        python3.10 -m venv venv
-        source venv/bin/activate
+        uv venv --python 3.10 --seed
```

Comment:
`--seed` is necessary to install `pip` into the new `venv`

Reply:
TIL!